doi/build.gradle

```diff
@@ -33,9 +33,11 @@ dependencies {
     implementation 'org.opencadc:cadc-access-control-identity:[1.1.0,)'
     implementation 'org.opencadc:cadc-vosi:[1.4.3,2.0)'
     implementation 'org.opencadc:cadc-vos:[2.0.7,3.0)'
+    implementation 'org.opencadc:cadc-uws:[1.0.5,)'
```

Comment:
Does doi build for you? I'm getting build errors for LocalAuthority, possibly from the reg update. Also Pat made an earlier comment that cadc-uws should not be needed in the build as he fixed some vos upstream dependencies.